### PR TITLE
NOC email template + deleteModal translations

### PR DESCRIPTION
**Pull request by minsunny**
nanopay : https://github.com/nanoPayinc/NANOPAY/pull/11878

https://nanopay.atlassian.net/browse/NP-3407
https://nanopay.atlassian.net/browse/NP-3564

added NOC template - NOC email notifications was used without no template, added basic templated to set its body and subject properly
DeleteModal translation - the 'label' was passed to deleteModal so it could not be translated properly.

### CODE REVIEW

File: src/foam/nanos/notification/EmailSetting.js [hunk 1]
```diff
@@ -91,13 +91,12 @@ foam.CLASS({
...
           }
-        }    
+        }
 
         try {
           if ( foam.util.SafetyUtil.isEmpty(notification.getEmailName()) ) {
-            message.setSubject(notification.getTemplate());
             message.setBody(notification.getBody());
-            EmailsUtility.sendEmailFromTemplate(x, null, message, null, null);
+            EmailsUtility.sendEmailFromTemplate(x, null, message, "NOCTemplate", null);
```
Comment (jlhughes) [inline src/foam/nanos/notification/EmailSetting.js]: Was this for testing?

Comment (minsunny) [reply src/foam/nanos/notification/EmailSetting.js]: added 'NOCTemplate' https://github.com/nanoPayinc/NANOPAY/pull/11878/files
to fix blank email notification, it happened on 'NOC' type notification that does not have a template.
so just added NOCTemplate - for no template cases

Comment (jlhughes) [reply src/foam/nanos/notification/EmailSetting.js]: I'm surprised to see we've lost our 'NOC' notification template.  Why not just create a notification template with the an email name? 

notification.template: 'NOC'
notification.emailName: 'NOC'

and then the corresponding emailTemplate, with id 'NOC'.

Comment (yij793) [reply src/foam/nanos/notification/EmailSetting.js]: I believe we do have "NOC" template which located in admin > notificationTemplates. that is the one it use.

Comment (jlhughes) [reply src/foam/nanos/notification/EmailSetting.js]: Yes, so that needs to be updated with a valid emailName - which is the emailTemplate.

**Comment by minsunny** [comment]
moved here and close this https://github.com/foam-framework/foam2/pull/4805

**Pull request closed without merging**